### PR TITLE
SISRP-27379 - add backToText to advisor resource and student resource links

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-27379

### CODE REVIEW

**Comment by pfarestveit** [comment]
@dfkaye - this is a functional change, right?  If so, it should be associated with a JIRA.

**Comment by dfkaye** [comment]
@pfarestveit -- Jira created -- let me know if any fields need correcting in it...

File: src/assets/javascripts/angular/controllers/widgets/studentResourcesController.js [hunk 1]
```diff
@@ -24,10 +20,14 @@ angular.module('calcentral.controllers').controller('StudentResourcesController'
    * Add the back to text
    */
   var addBackToText = function(resources) {
```
Comment (dfkaye) [inline src/assets/javascripts/angular/controllers/widgets/studentResourcesController.js]: @redconfetti - I'll let you make that call - this stuff should go in 7.2 soon, though

Comment (redconfetti) [inline src/assets/javascripts/angular/controllers/widgets/studentResourcesController.js]: Spoke with @dfkaye and decided that I'll make a follow up PR to implement these changes.

Comment (redconfetti) [inline src/assets/javascripts/angular/controllers/widgets/studentResourcesController.js]: PR #6165 implements the refactor mentioned above

**Pull request merged**